packages/features/settings/CreateOrEditOutOfOfficeModal.tsx

```diff
@@ -25,17 +26,34 @@ export const CreateOrEditOutOfOfficeEntryModal = ({
   openModal,
   closeModal,
   currentlyEditingOutOfOfficeEntry,
+  oooType,
+  setTeamOOOEntriesUpdated,
+  setMyOOOEntriesUpdated,
 }: {
   openModal: boolean;
   closeModal: () => void;
   currentlyEditingOutOfOfficeEntry: BookingRedirectForm | null;
+  oooType: string;
+  setTeamOOOEntriesUpdated: Dispatch<SetStateAction<number>>;
+  setMyOOOEntriesUpdated: Dispatch<SetStateAction<number>>;
 }) => {
   const { t } = useLocale();
   const utils = trpc.useUtils();
 
   const { data: listMembers } = trpc.viewer.teams.listMembers.useQuery({});
   const me = useMeQuery();
-  const memberListOptions: {
+  const forwardingToMemberListOptions: {
+    value: number;
+    label: string;
+  }[] =
+    listMembers
+      ?.filter((member) => (oooType === "mine" ? me?.data?.id !== member.id : oooType === "team"))
```

Comment:
It’s better to use enum instead of hardcoded values like "mine" and "team."

Reply:
ok